### ISSUE TITLE
No framework for defining help entries

### ISSUE THREAD

**Issue by PrinterElf** [hangoutsbot/hangoutsbot#703]
The help entries for many of the plugins are somewhat hit and miss.
It would be desirable to have a framework from which the help entries for plugins can be copied and formatted consistently across the user experience.

**Comment by PrinterElf** [comment]
Unsure if this should be an extension to Readme.md, or an amendment to the Plugin Authoring section of the Wiki.

**Comment by endofline** [comment]
might be possible to insert a link into the docstring for <further info> - will consider for addition into v3 framework

**Comment by PrinterElf** [comment]
I think there may need to be some differentiation between the simple doc text in the file, and the help text output on user request.
The latter is likely to need to be far more verbose than the former, especially with the myriad of sub commands.
tldr for example appears as just one command in the "/bot help" list, but has add, retrieve, edit, delete as sub commands/functions, all of which need explaining to the user with syntax.

**Comment by PrinterElf** [comment]
> further consideration: language differences for localised bots will make this difficult to implement in any consistent way

What is the string definition for localised bots?
i.e. How are the text strings identified as locale customisable entities, and how/where are they presently handled?
